src/descriptor/policy.rs

```diff
@@ -886,43 +911,76 @@ impl<Ctx: ScriptContext> ExtractPolicy for Miniscript<DescriptorPublicKey, Ctx>
     }
 }
 
+/// Options to build the satisfaction field in the policy
+#[derive(Debug, Clone, Copy)]
+pub enum BuildSatisfaction<'a> {
+    /// Don't generate `satisfaction` field
+    None,
+    /// Analyze the given PSBT to check for existing signatures
+    Psbt(&'a PSBT),
+    /// Like `Psbt` variant and also check for expired timelocks
+    PsbtTimelocks {
+        /// give PSBT
```

Comment:
nit, first letter capital and "Given PSBT"

Reply:
fixed in 6628a685ce9995c2faddbae88c1e1baf322491b2